recipes-core/1u0022-setup-hostname/1u0022-setup-hostname/set_hostname.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/sh
+
+if [ -f /srv/hostname ]; then
+    NEW_HOSTNAME=$(cat /srv/hostname)
+else
+    echo "No hostname file found in /srv/hostname"
+    exit 1
+fi
+
+echo $NEW_HOSTNAME > /tmp/hostname
```

Comment:
you should not use `/tmp` here; `/run` would be a better location.

Or (when the bind mounting is really required): why use a temporary file instead of bind-mounting `/srv/hostname` directly?

Reply:
Thanks for your input! Yes, bind mounting /srv/hostname directly makes sense. 
We couldn't find an other way to set the hostname without making /etc/hostname rw. Is there a more elegant way to solve this?